src/main/kotlin/no/nav/familie/ks/sak/kjerne/avstemming/GrensesnittavstemmingScheduler.kt

```diff
@@ -50,7 +51,7 @@ class GrensesnittavstemmingScheduler(
             val sisteFerdigTaskData =
                 objectMapper.readValue(sisteFerdigTask.payload, GrensesnittavstemmingTaskDto::class.java)
             fom = sisteFerdigTaskData.tom.toLocalDate().atStartOfDay()
-            tom = nesteGyldigeTriggertidForBehandlingIHverdager((24 * 60).toLong(), fom).toLocalDate().atStartOfDay()
+            tom = utledNesteTriggerTidIHverdagerForTask(fom, Duration.ofMinutes(24 * 60)).toLocalDate().atStartOfDay()
```

Comment:
Kan kanskje bare gjøre dette til `Duration.ofHours(24)`? 

```suggestion
            tom = utledNesteTriggerTidIHverdagerForTask(fom, Duration.ofHours(24)).toLocalDate().atStartOfDay()
```